### PR TITLE
🔧 collection info royalty

### PR DESCRIPTION
**Pull request by roiLeo**
- fix issue on colleciton infos when royalty is set on evm address
- fix i18n warning

Tested on `/ahp/collection/146`

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [ ] Closes #

## Screenshot 📸

**before**
Capture d’écran 2024-06-18 à 11 25 19 AM

**after**
Capture d’écran 2024-06-18 à 11 22 11 AM

Screenshot 2024-06-18 at 11-22-51 Block DJ One Stop Shop for Polkadot NFTs

### CODE REVIEW

File: components/collection/CollectionInfo.vue [hunk 1]
```diff
@@ -28,7 +28,11 @@
             <template #content>
               <div class="bg-background-color border py-2 px-4 text-xs">
                 Recipient Address:
+                <div v-if="recipient.startsWith('0x')" class="break-all">
```
Comment (preschian) [inline components/collection/CollectionInfo.vue:31]: I believe we already had some utils to check if the address is on evm. CMIIW
cc @hassnian

**Review by preschian — COMMENTED** [review]
hi, long time no see 👋🏻

**Comment by vikiival** [comment]
I believe that input address is incorrect

File: components/collection/CollectionInfo.vue (hunk 1 above)
Comment (vikiival) [reply components/collection/CollectionInfo.vue:31]: `import { isEthereumAddress } from '@polkadot/util-crypto'`

**Review by vikiival — CHANGES_REQUESTED** [review]
https://github.com/kodadot/nft-gallery/pull/10482#discussion_r1645763637

**Comment by vikiival** [comment]
tbh this is just a random string :)

public key on polkadot has - 66 chars (0x + 64)
ETH address has - 42 chars (0x + 40)

File: components/collection/CollectionInfo.vue [hunk 2]
```diff
@@ -29,13 +29,17 @@
               <div class="bg-background-color border py-2 px-4 text-xs">
                 Recipient Address:
                 <nuxt-link
+                  v-if="isValidAddress(recipient)"
```
Comment (vikiival) [inline components/collection/CollectionInfo.vue:32]: but valid address is also evm address on Base

Comment (roiLeo) [reply components/collection/CollectionInfo.vue:32]: then what should it link to ?

Comment (vikiival) [reply components/collection/CollectionInfo.vue:32]: I am just telling that valid address depends on chain prefix.

Comment (roiLeo) [reply components/collection/CollectionInfo.vue:32]: What solution do you suggest?

**Review by hassnian — COMMENTED** [review]
> make a function isAddressValid(adddress, prefix),
based on The VM type of the prefix use correct function to check

check `/utils/vm.ts` `execByVm` you can pass a `prefix` param

**Comment by vikiival** [comment]
This will be probably fixer by release of - https://github.com/kodadot/stick/releases/tag/v13.1

**Comment by roiLeo** [comment]
Same issue on `/ahp/collection/93`

**Comment by vikiival** [comment]
> Same issue on `/ahp/collection/93`

- https://github.com/kodadot/stick/commit/efe6c5dc6937638bff09ff4cca5daa12fafd1ac3

**Pull request closed without merging**